lib/Backend/IRBuilderAsmJs.cpp

```diff
@@ -2553,6 +2553,10 @@ IRBuilderAsmJs::BuildInt2(Js::OpCodeAsmJs newOpcode, uint32 offset, Js::RegSlot
         }
 
         break;
+    
+    case Js::OpCodeAsmJs::Eqz_Int:
+        instr = IR::Instr::New(Js::OpCode::CmEqz_I4, dstOpnd, srcOpnd, m_func);
```

Comment:
Do we really need a new opcode for this?  I think cmeq_i4 with opnd2 of 0 should be fine